daemon/config/config_test.go

```diff
@@ -374,6 +397,17 @@ func TestValidateConfiguration(t *testing.T) {
 				},
 			},
 		},
+		{
+			config: &Config{
+				CommonConfig: CommonConfig{
+					MaxDownloadAttempts: &minusNumber,
```

Comment:
I was confused why this passed, but looks like `minusNumber` here, is actually a _positive_ number.

Changing to use a utility function (as I commented on the other test above) would prevent such confusions/mistakes